pkgs/by-name/ff/ffsubsync/package.nix

```diff
@@ -15,10 +16,12 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-ZdKZeKfAUe/FXLOur9Btb5RgXewmy3EHunQphqlxpIc=";
   };
 
-  nativeBuildInputs = with python3.pkgs; [
-    setuptools
+  patches = [
+    ./python312.patch # currently not applied in any released version
```

Comment:
Why not `fetchpatch` https://github.com/smacke/ffsubsync/commit/de75bdbfe846b3376f8c0bcfe2e5e5db82d7ff20 ?

Reply:
Completely forgot it existed, it makes a whole lot more sense here, yes. Fixed.